# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/rivatv/module-install-path.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.in	2004-03-15 21:21:17.000000000 +0100
+++ ./Makefile.in.patch	2004-08-02 12:08:40.000000000 +0200
@@ -17,11 +17,11 @@
 
 
 # Location of your module tree
-MODULES	     = /lib/modules/@UTS_RELEASE@/kernel
+MODULES	     = /lib/modules/@UTS_RELEASE@
 
 
 # Destination directory of the built modules within $(MODULES).
-MOD_DEST     = /drivers/media/video
+MOD_DEST     = /rivatv
 
 #=============================================================================
 #
